packages/typespec-python/generator/pygen/black.py

```diff
@@ -37,14 +37,14 @@ def process(self) -> bool:
                 [
                     Path(f)
                     for f in self.list_file()
-                    if all(
-                        item not in f
-                        for item in (
-                            "__pycache__",
-                            "node_modules",
-                            ".tox",
-                            ".mypy_cache",
-                        )
+                    if Path(f).parts[0]
+                    not in (
+                        "__pycache__",
+                        "node_modules",
+                        ".tox",
+                        ".mypy_cache",
+                        "venv",
+                        "env",
```

Comment:
what about `.venv`?

Reply:
I think usually virtual environment will not be named with `.venv`

Reply:
Add new rule to skip folder that starts with `.`